hubspot/management/commands/configure_hubspot_bridge.py

```diff
@@ -87,7 +87,19 @@
                 "fieldType": "text",
             },
         ],
-    }
+    },
+    "contacts": {
```

Comment:
Also include the following here or else the `configure_hubspot_bridge` command will fail:
```
"groups": [],
```

Reply:
added